Makefile

```diff
@@ -1,18 +1,20 @@
 # Set DESTDIR if it isn't given
 DESTDIR?=/
 
-.PHONY : all clean clean-build clean-pyc clean-test install test upload
+.PHONY : all clean clean-build clean-pyc clean-test install test upload develop
 
 all :
 	python ./setup.py build
 
 install : all
 	python ./setup.py install --root=${DESTDIR}
 
+develop :
```

Comment:
I prefer removing it.
I think that a developer won't necessary known this make target but will probably known the python setup.py develop command and will run it directly.